static/js/project/controllers/project.controller.js

```diff
@@ -249,5 +256,83 @@
           }
         }
       }
-  }
+
+      // watch filesToUpload and upload multiple files in any format by calling uploadViaFileManager
+      $scope.$watch(
```

Comment:
pass the third argument as true and you don't need to return the function, that also does a deep watch